man/rgbasm.5

```diff
@@ -1104,7 +1104,9 @@ Additionally, label names can contain up to a single dot
 .Ql \&. ,
 which may not be the first character.
 .Pp
-A symbol cannot have the same name as a reserved keyword, unless it is prefixed by a hash
+A symbol cannot have the same name as a reserved keyword, unless it is a
+.Dq raw symbol
+prefixed by a hash
 .Sq # .
```

Comment:
I think this should be more explicit that “raw symbol” means “prefixed with a hash”.

I'm also realising that “raw identifier” is a better name, since it might not be referring to a symbol yet? (And identifiers are *names* for symbols. I'd be fine with calling those “raw names”, if that sounds better than “identifier”.)